### ISSUE TITLE
macOS missing dependency: ModuleNotFoundError: No module named 'screeninfo'

### ISSUE THREAD

**Issue by williamrexhardin** [pokepetter/ursina#650]
Fresh 3.9.6 venv && pip install ursina

`import ursina`  fails with the following:

> macOS missing dependency: ModuleNotFoundError: No module named 'screeninfo'
 
 After pip installing screeninfo, the import works. Seems like `screeninfo` needs to be added to requirements.txt for Ursina usage on darwin platforms, perhaps it used to be transitively installed but got dropped somewhere. If I keep poking around and like the engine I'll raise a PR.

**Comment by pokepetter** [comment]
It is listed in setup.py. Maybe the installation failed at some point?

**Comment by williamrexhardin** [comment]
@pokepetter  Looks like `pip install ursina` resolves 5.2.0 and not the latest version listed on pypi. Looks like versions 

`pip download --no-cache-dir --verbose ursina` 

```
  Link requires a different Python (3.9.6 not in: '>=3.10'): https://files.pythonhosted.org/packages/32/16/28334b236ad0531d83fcf3ac464ce7ed699f49a236155b3994e5dc0b055b/ursina-5.3.0-py3-none-any.whl#sha256=4de36ed2e85f9455d6e321e3974268cefec2361ee87c301b9afb81c5ac328904 (from https://pypi.org/simple/ursina/) (requires-python:>=3.10)
  Link requires a different Python (3.9.6 not in: '>=3.10'): https://files.pythonhosted.org/packages/52/c4/3e8ad3061eafd8ca24c325a8218f4c94255aa21844a8857ad4f6bb564181/ursina-5.3.0.tar.gz#sha256=3acabaef9f2630feb90d1f1d2c847e86f7a65acdd0beb8194c89ececa6112534 (from https://pypi.org/simple/ursina/) (requires-python:>=3.10)
  Link requires a different Python (3.9.6 not in: '>=3.10'): https://files.pythonhosted.org/packages/db/a8/97b3fd9aac086c1d00a6b7dca593570f0fb3cc385c778df9e6df1cce7299/ursina-6.0.0-py3-none-any.whl#sha256=855f8df6dbcb5c25634c1565788fd128dd538b26a23ac24ec0624620b26fc207 (from https://pypi.org/simple/ursina/) (requires-python:>=3.10)
  Link requires a different Python (3.9.6 not in: '>=3.10'): https://files.pythonhosted.org/packages/21/bb/0537acd91717d9e81c44843d651a450072bc8332e2ae466c91d82bb751e3/ursina-6.0.0.tar.gz#sha256=c427090b4f60cb7dcb52c02966033286a6598facbdd5d41940f2052a3dbaffd7 (from https://pypi.org/simple/ursina/) (requires-python:>=3.10)
  Link requires a different Python (3.9.6 not in: '>=3.10'): https://files.pythonhosted.org/packages/3d/5b/b0336f5731b87192d001fde6ae5d0a96946101a38d20f06e749145c26ef7/ursina-6.1.0-py3-none-any.whl#sha256=7ca67a052ca7f495a8fa368273219b727fac6e5afabf9b74e08d35c8f2a22416 (from https://pypi.org/simple/ursina/) (requires-python:>=3.10)
  Link requires a different Python (3.9.6 not in: '>=3.10'): https://files.pythonhosted.org/packages/88/92/ef636d54269dfea0b7fed92000c8104466ec7625b44a7a8ffe820383408d/ursina-6.1.0.tar.gz#sha256=e962d55711dba6679ccc4c5d3afb3134cdf007e02c8e251eb8c094823bd34f3c (from https://pypi.org/simple/ursina/) (requires-python:>=3.10)
  Link requires a different Python (3.9.6 not in: '>=3.10'): https://files.pythonhosted.org/packages/de/64/54ea475b7f550b6586e76e77b5c793f76fb7cc3a3f0fe74e3af6dd5f0a67/ursina-6.1.1-py3-none-any.whl#sha256=6a78dc182c2c97d6b1e8648013b542e7ed4208ceb8d7f66da516e511fcda7ac3 (from https://pypi.org/simple/ursina/) (requires-python:>=3.10)
  Link requires a different Python (3.9.6 not in: '>=3.10'): https://files.pythonhosted.org/packages/cd/a1/5b04f371264d4796cf387905f343595efed94a4261821f82a39cbdbf78ee/ursina-6.1.1.tar.gz#sha256=266992f24c29359cbe1c9f15dc02a70a1333efabffffb42436fa2a4f53a5c1fd (from https://pypi.org/simple/ursina/) (requires-python:>=3.10)
  Link requires a different Python (3.9.6 not in: '>=3.10'): https://files.pythonhosted.org/packages/2c/ca/15c417ef8cd0d35a9911b4f32d6470c32b5806352ed1eba1a32489a8dd14/ursina-6.1.2-py3-none-any.whl#sha256=7dcf298ef558b0c31f27ce793f53bdada82c124989613df2ecc65a05effff3a5 (from https://pypi.org/simple/ursina/) (requires-python:>=3.10)
  Link requires a different Python (3.9.6 not in: '>=3.10'): https://files.pythonhosted.org/packages/1a/53/e4c1cb4005741860966e489e8672109149a07e874edd91deeb55c951d315/ursina-6.1.2.tar.gz#sha256=b76718e338fe7a6090946beed5de2daea23808b6e0c96945e2fb082025c036e6 (from https://pypi.org/simple/ursina/) (requires-python:>=3.10)
Collecting ursina
  Downloading ursina-5.2.0-py3-none-any.whl (2.4 MB)
     |████████████████████████████████| 2.4 MB 468 kB/s
     
 ```

**Comment by pokepetter** [comment]
As the error says, you need Python version 3.10 or higher. Most of the things in ursina will work with older versions, but newer additions to the engine, like the level editor uses some of the new syntax.
